docs_src/pipelinerun.md

```diff
@@ -3,122 +3,134 @@
 
-#### For Core systems
-```
-sudo ./docker-run.sh --platform core --inputsrc <serial_number> --realsense_enabled
+### `--realsense_enabled`
+TO use realsense camera and to provide realsense camera 12 digit serial number as inputsrc to the docker-run.sh script
 
-```
-Obtaining RealSense camera serial number: [How_to_get_serial_number](./camera_serial_number.md)
+### `--barcode`
+To provide barcode detection frame internal value such as `--barcode 5`, default recognition interval value is 5
 
-### Option 4 to run with video file input:
+### `--color-width`
+Realsense camera color related property, to apply realsense camera color width, which will overwrite the default value of realsense gstreamer; if it's not provided, it will use the default value from realsense gstreamer; make sure to look up to your realsense camera's color capability using `rs-enumerate-devices`
 
-#### For Core systems
-```
-sudo ./docker-run.sh --platform core --inputsrc file:my_video_file.mp4
-```
+### `--color-height`
+Realsense camera color related property, to apply realsense camera color height, which will overwrite the default value of realsense gstreamer; if it's not provided, it will use the default value from realsense gstreamer; make sure to look up to your realsense camera's color capability using `rs-enumerate-devices`
 
-#### For Xeon systems
-```
-sudo ./docker-run.sh --platform xeon --inputsrc file:my_video_file.mp4
-```
+### `--color-framerate`
+Realsense camera color related property, to apply realsense camera color framerate, which will overwrite the default value of realsense gstreamer; if it's not provided, it will use the default value from realsense gstreamer; make sure to look up to your realsense camera's color capability using `rs-enumerate-devices`
 
-#### For Discrete GPU systems
+---
+## RealSense pipeline run with optional parameters script example:
+```bash
+sudo ./docker-run.sh --platform core --inputsrc <serial_number> --realsense_enabled --color-width 1920 --color-height 1080 --color-framerate 15 --ocr 5 CPU
 ```
-sudo ./docker-run.sh --platform dgpu.x --inputsrc file:my_video_file.mp4 --ocr 5 GPU
-```
-Note: for `dgpu.x`, the x can be 0, 1, 2 depends on how many discrete gpu it has in the system
-
-
+---
 ## Check for pipeline run success 
 
 Make sure the command was successful. To do so, run:
 
-```
+```bash
 docker ps --format 'table{{.Image}}\t{{.Status}}\t{{.Names}}'
 ```
 
 !!! Successful Results
-    Your output for Core is as follows:
 
-    | IMAGE                                              | STATUS                   | NAMES                 |
-    | -------------------------------------------------- | ------------------------ |-----------------------|
-    | sco-soc:2.0                                        | Up 9 seconds             | vision-self-checkout0 |
+Your output for Core is as follows:
```

Comment:
can these two tables be combined? something like:

Your output for each type should be as follows
| Platform Type | IMAGE | STATUS | NAMES |

Reply:
this was meant for `docker ps` output as how it is shown in terminal return, I think if I add the `platform type` in the table may confuse user.